features/steps/command/shell.feature

```diff
@@ -0,0 +1,76 @@
+Feature: Running shell commands
+
+  In order to test interactive command line applications
```

Comment:
I used that style myself in some features, but move away from this:

We want to generate documentation from the feature files in the near future. So please write some more "prosa" in natural language you would normally expect in documentation.

See https://github.com/cucumber/aruba/blob/master/features/api/command/run.feature for an example. Hope it's clear what I mean.